code/modules/multiz/movement.dm

```diff
@@ -376,60 +379,32 @@
 					"You hear something slam into \the [landing].")
 			playsound(loc, "punch", 25, 1, -1)
 
-		if(planetary)	//Since the planetary fall damage is calibrated for humans, we need to up this a bit
-			damage_min *= 2
-			damage_max *= 2
-
-		adjustBruteLoss(rand(damage_min, damage_max))
+		// Because wounds heal rather quickly, 10 (the default for this proc) should be enough to discourage jumping off but not be enough to ruin you, at least for the first time.
+		// Hits 10 times, because apparently targeting individual limbs lets certain species survive the fall from atmosphere
+		for(var/i = 1 to 10)
+			adjustBruteLoss(rand(damage_min, damage_max))
```

Comment:
Does this increase the damage of regular falls (IE from deck to deck) by ten times?

Reply:
Should actually be slightly weaker.
Old proc added that amount of damage to every limb on the body.